### DISCUSSION TITLE
about table cell edit doc

### DISCUSSION THREAD

**Discussion by sonicwong** [primefaces/community#303] [PrimeNG]
https://primeng.org/table#cell-edit
The doc said using `[pEditableColumn]="product.code"`,
but I found data is missing in $event obj when added `(onEditComplete)="onEditDone($event)"` to `p-table`.
If change to `[pEditableColumn]="product"`, data returned.
Is it wrong in doc?

```
<td [pEditableColumn]="product.code" pEditableColumnField="code">
                <p-cellEditor>
                    <ng-template pTemplate="input">
                        <input pInputText type="text" [(ngModel)]="product.code" />
                    </ng-template>
                    <ng-template pTemplate="output">
                        {{ product.code }}
                    </ng-template>
                </p-cellEditor>
            </td>

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a Stackblitz example so we can identify the issue clearly?
